test/Sentry.Tests/SentryHttpMessageHandlerTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NSubstitute;
+using Sentry.Protocol;
+using Sentry.Testing;
+using Xunit;
+
+namespace Sentry.Tests
+{
+    public class SentryHttpMessageHandlerTests
+    {
+        [Fact]
+        public async Task SendAsync_SentryTraceHeaderNotSet_SetsHeader()
```

Comment:
1 test can really cover all edge cases this code can have?

Reply:
2 tests for 2 branches, so I can't think of anything else ;)